src/Analyzers/CSharp/CodeFixes/UseSimpleUsingStatement/UseSimpleUsingStatementCodeFixProvider.cs

```diff
@@ -86,27 +87,25 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             return currentBlock;
         }
 
-        private static IEnumerable<StatementSyntax> Expand(UsingStatementSyntax usingStatement)
+        private static ImmutableArray<StatementSyntax> Expand(UsingStatementSyntax usingStatement)
```

Comment:
The linked issue is about trivia in simplify if/else - did you mean to link a different issue?

Reply:
> The linked issue is about trivia in simplify if/else - did you mean to link a different issue?

yes... now i have to find that...

Reply:
https://github.com/dotnet/roslyn/issues/58861!